### PR TITLE
Add unmatched bracket highlighting capability

### PR DESCRIPTION
**Pull request by jnsebgosselin**
This PR is to add the capability to highlight unmatched bracket in red in the console.

ipython_unmatched

### CODE REVIEW

File: qtconsole/bracket_matcher.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def __init__(self, text_edit):
         self.format = QtGui.QTextCharFormat()
         self.format.setBackground(QtGui.QColor('silver'))
 
+        self.format_unmatched = QtGui.QTextCharFormat()
+        self.format_unmatched.setBackground(QtGui.QColor('red'))
```
Comment (wmvanvliet) [inline qtconsole/bracket_matcher.py]: Can we make this color configurable in the CSS stylesheet? It may clash with someone's colorscheme.

**Comment by JeremySikes73** [comment]
@wmvanvliet, #253 is meant to implement this feature more thoroughly, instead of this PR.

**Comment by wmvanvliet** [comment]
@jnsebgosselin are you ok with @JeremySikes73 taking this over?

**Comment by jnsebgosselin** [comment]
@wmvanvliet sure, as long as backward compatibility is kept with Spyder, I'm glad someone can take this over and bring it a step further.

**Pull request closed without merging**